src/main/scala/io/archivesunleashed/app/WriteGraphXML.scala

```diff
@@ -0,0 +1,83 @@
+/*
```

Comment:
I don't understand how this file is part of this PR?
It's already checked in? https://github.com/archivesunleashed/aut/blob/master/src/main/scala/io/archivesunleashed/app/WriteGraphML.scala

Reply:
This file is different because it creates GraphMl file from GraphX object not a simple RDD. Also, it has an extra field for pageRank. Other fields will be added as the code progresses.